src/app/containers/edit-rubric/edit-rubric.component.html

```diff
@@ -4,7 +4,7 @@
             <rc-edit-rubric-header></rc-edit-rubric-header>
             <div class="row">
                 <div class="table-responsive">
-                    <table class="table table-striped">
+                    <table class="table table-striped" aria-label="Create Rubric Body">
```

Comment:
This is a great idea. Rather than call it "Create Rubric Body", I might be more specific and make it "Create Rubric Components and Levels".

"Body" could be interpreted as a lot of things, while most teachers will know what "Rubric Components" and "Rubric Levels" are.